pkg/webhook/uniteddeployment/validating/uniteddeployment_validation.go

```diff
@@ -380,6 +395,14 @@ func validateDeployment(deployment *appsv1alpha1.DeploymentTemplateSpec, fldPath
 
 	return allErrs
 }
+func validateCloneSet(cs *appsv1alpha1.CloneSetTemplateSpec, fldPath *field.Path) field.ErrorList {
```

Comment:
The check is too simple, is it possible to refer to the cloneset validate webhook?